src/playground/src/ui/screen/showcases/select/index.ts

```diff
@@ -4,3 +4,7 @@ export { SelectWithGroupsShowcase } from './selectWithGroups.component';
 export { SelectCustomIconShowcase } from './selectCustomIcon.component';
 export { SelectStatusShowcase } from './selectStatus.component';
 export { SelectInlineStylingShowcase } from './selectInlineStyling.component';
+export { SelectPreselectedInlineShowcase } from './selectPreselectedInline.component';
```

Comment:
Let's leave only this one for docs and call it `SelectInitialValueShowcase`

Reply:
will do